src/gradata/_encryption.py

```
@@ -116,4 +116,4 @@ def close_encrypted_db(brain_dir: Path, encryption_key: str) -> None:
         db_path.unlink()
     else:
         tmp_path.unlink(missing_ok=True)
-        raise RuntimeError("Encryption produced empty or missing file — plaintext preserved")
\ No newline at end of file
+        raise RuntimeError("Encryption produced empty or missing file — plaintext preserved")
```

Ruff (0.15.9)

[warning] Avoid specifying long messages outside the exception class

(TRY003)